test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -793,4 +785,142 @@ public void sealDoesNotModifyClientRouterEpoch() throws Exception {
         }
 
     }
+
+    @Test
+    public void updateTrailingLayoutServers() throws Exception {
+        addServer(SERVERS.PORT_0);
+        addServer(SERVERS.PORT_1);
+        addServer(SERVERS.PORT_2);
+
+        Layout layout = new TestLayoutBuilder()
```

Comment:
can we not have simple short methods for all this boilerplate. This way it is easier to read and also easier to write more variations of the test.

Reply:
Done